Build failure: limine: enableAll not working since recent 9.5.2 -> 9.6.1 bump

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Since #434161, which bumped from 9.5.2 to 9.6.1 (yes, the title indicating 9.6.0 is misleading), the `enableAll` build is broken. You can test using:

`nix-build -E "let pkgs = import ./default.nix {}; in pkgs.limine.override { enableAll = true; }`


### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
64/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/e820.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/idt.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/idt.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/lapic.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/pic.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/smp.o' '/build/limine-9.6.1/common-uefi-x86-64/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-uefi-x86-64/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-uefi-x86-64/nyu-efi/x86_64/reloc.o' '/build/limine-9.6.1/common-uefi-x86-64/nyu-efi/x86_64/entry.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/mem.o' '/build/limine-9.6.1/common-uefi-x86-64/common/menu_thunk.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-uefi-x86-64/common/efi_thunk.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/spinup.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux_64.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/limine_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/smp_trampoline.o' -o '/build/limine-9.6.1/common-uefi-x86-64/limine_nomap.elf'
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-nostdlibinc' [-Wunused-command-line-argument]
cd '/build/limine-9.6.1/common-uefi-x86-64' && \
        '/build/limine-9.6.1/common/gensyms.sh' '/build/limine-9.6.1/common-uefi-x86-64/limine_nomap.elf' full 64 '\.text'
clang -target x86_64-unknown-none-elf -g -O2 -pipe -g -Wall -Wextra -Wshadow -Wvla -Wno-error -std=gnu11 -nostdinc -ffreestanding -ffunction-sections -fdata-sections -fno-stack-protector -fno-stack-check -fno-omit-frame-pointer -fno-strict-aliasing -fno-lto -fPIE -fshort-wchar -m64 -march=x86-64 -mabi=sysv -mno-80387 -mno-mmx -mno-sse -mno-sse2 -mno-red-zone -I ../nyu-efi/inc -I . -I libc-compat -I ../limine-protocol/include -I ../flanterm/src -I ../libfdt/src -I '/build/limine-9.6.1/common-uefi-x86-64/..' -isystem ../freestnd-c-hdrs/include  -DCOM_OUTPUT=false -DE9_OUTPUT=false -DLIMINE_API_REVISION=3 -DFLANTERM_IN_FLANTERM -MMD -MP -DUEFI -c '/build/limine-9.6.1/common-uefi-x86-64/full.map.S' -o '/build/limine-9.6.1/common-uefi-x86-64/full.map.o'
Warning: supplying the --target x86_64-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
rm -f '/build/limine-9.6.1/common-bios/stage2.map.S' '/build/limine-9.6.1/common-bios/stage2.map.d'
make -f common.mk '/build/limine-9.6.1/common-bios/empty'
make[3]: Entering directory '/build/limine-9.6.1/common'
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-nostdlibinc' [-Wunused-command-line-argument]
make[3]: '/build/limine-9.6.1/common-bios/empty' is up to date.
make[3]: Leaving directory '/build/limine-9.6.1/common'
make -f common.mk '/build/limine-9.6.1/common-bios/linker_nomap.ld'
make[3]: Entering directory '/build/limine-9.6.1/common'
rm -f '/build/limine-9.6.1/common-uefi-x86-64/full.map.S' '/build/limine-9.6.1/common-uefi-x86-64/full.map.d'
make -f common.mk '/build/limine-9.6.1/common-uefi-x86-64/linker.ld'
make[4]: Entering directory '/build/limine-9.6.1/common'
/nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p '/build/limine-9.6.1/common-bios'
clang -target i686-unknown-none-elf -target i686-unknown-none-elf -x c -E -P -undef -DLINKER_NOMAP linker_bios.ld.in -o '/build/limine-9.6.1/common-bios/linker_nomap.ld'
Warning: supplying the --target i686-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
/nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p '/build/limine-9.6.1/common-uefi-x86-64'
clang -target x86_64-unknown-none-elf -target x86_64-unknown-none-elf -x c -E -P -undef linker_uefi_x86_64.ld.in -o '/build/limine-9.6.1/common-uefi-x86-64/linker.ld'
Warning: supplying the --target x86_64-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
make[3]: Leaving directory '/build/limine-9.6.1/common'
ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' -T'/build/limine-9.6.1/common-bios/linker_nomap.ld' -o '/build/limine-9.6.1/common-bios/limine_nos3map.elf'
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
make[4]: Leaving directory '/build/limine-9.6.1/common'
ld.lld \
        -T'/build/limine-9.6.1/common-uefi-x86-64/linker.ld' \
        -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_x86_64 -pie -z text '/build/limine-9.6.1/common-uefi-x86-64/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/crypt/blake2b.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/edid.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/gop.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/serial.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/vbe.o' '/build/limine-9.6.1/common-uefi-x86-64/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-uefi-x86-64/common/entry.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/entry.s3.o' '/build/limine-9.6.1/common-uefi-x86-64/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/fs/file.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/acpi.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/bli.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/config.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/elf.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/elsewhere.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/fb.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/fdt.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/getchar.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/gterm.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/guid.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/image.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/libc.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/libc.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/memory.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/misc.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/misc.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/panic.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/part.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/pe.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/print.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/rand.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/stb_image.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/term.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/time.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/trace.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/uri.o' '/build/limine-9.6.1/common-uefi-x86-64/common/menu.o' '/build/limine-9.6.1/common-uefi-x86-64/common/mm/mtrr.o' '/build/limine-9.6.1/common-uefi-x86-64/common/mm/pmm.o' '/build/limine-9.6.1/common-uefi-x86-64/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/mm/vmm.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/chainload.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/limine.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux_risc.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot1.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/a20.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/e820.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/idt.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/idt.s2.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/lapic.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/pic.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/smp.o' '/build/limine-9.6.1/common-uefi-x86-64/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-uefi-x86-64/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-uefi-x86-64/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-uefi-x86-64/nyu-efi/x86_64/reloc.o' '/build/limine-9.6.1/common-uefi-x86-64/nyu-efi/x86_64/entry.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/mem.o' '/build/limine-9.6.1/common-uefi-x86-64/common/menu_thunk.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-uefi-x86-64/common/efi_thunk.o' '/build/limine-9.6.1/common-uefi-x86-64/common/lib/spinup.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux_64.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/limine_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/linux_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-uefi-x86-64/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-uefi-x86-64/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-uefi-x86-64/full.map.o' -o '/build/limine-9.6.1/common-uefi-x86-64/limine.elf'
llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine_nos3map.elf' '/build/limine-9.6.1/common-bios/build-id.s2.bin'
cd '/build/limine-9.6.1/common-bios' && \
        llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s2.bin build-id.s2.o && \
        llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s2.o
llvm-objcopy -O binary '/build/limine-9.6.1/common-uefi-x86-64/limine.elf' '/build/limine-9.6.1/common-uefi-x86-64/BOOTX64.EFI'
chmod -x '/build/limine-9.6.1/common-uefi-x86-64/BOOTX64.EFI'
dd if=/dev/zero of='/build/limine-9.6.1/common-uefi-x86-64/BOOTX64.EFI' bs=4096 count=0 seek=$(( ($(wc -c < '/build/limine-9.6.1/common-uefi-x86-64/BOOTX64.EFI') + 4095) / 4096 )) 2>/dev/null
make[3]: Leaving directory '/build/limine-9.6.1/common'
make[2]: Leaving directory '/build/limine-9.6.1'
make '/build/limine-9.6.1/bin/BOOTX64.EFI'
make[2]: Entering directory '/build/limine-9.6.1'
llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine_nos3map.elf' '/build/limine-9.6.1/common-bios/build-id.s3.bin'
/nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p '/build/limine-9.6.1/bin'
cp '/build/limine-9.6.1/common-uefi-x86-64/BOOTX64.EFI' '/build/limine-9.6.1/bin/'
make[2]: Leaving directory '/build/limine-9.6.1'
cd '/build/limine-9.6.1/common-bios' && \
        llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s3.bin build-id.s3.o && \
        llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s3.o
ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' '/build/limine-9.6.1/common-bios/build-id.s2.o' '/build/limine-9.6.1/common-bios/build-id.s3.o' -T'/build/limine-9.6.1/common-bios/linker_nomap.ld' -o '/build/limine-9.6.1/common-bios/limine_nos3map.elf'
cd '/build/limine-9.6.1/common-bios' && \
        '/build/limine-9.6.1/common/gensyms.sh' '/build/limine-9.6.1/common-bios/limine_nos3map.elf' full 32 '\.text'
clang -target i686-unknown-none-elf -g -O2 -pipe -g -Wall -Wextra -Wshadow -Wvla -Wno-error -std=gnu11 -nostdinc -ffreestanding -ffunction-sections -fdata-sections -fno-stack-protector -fno-stack-check -fno-omit-frame-pointer -fno-strict-aliasing -fno-lto -fno-PIC -m32 -march=i686 -mabi=sysv -mno-80387 -mno-mmx -I . -I libc-compat -I ../limine-protocol/include -I ../flanterm/src -I ../libfdt/src -I '/build/limine-9.6.1/common-bios/..' -isystem ../freestnd-c-hdrs/include  -DCOM_OUTPUT=false -DE9_OUTPUT=false -DLIMINE_API_REVISION=3 -DFLANTERM_IN_FLANTERM -MMD -MP -DBIOS -c '/build/limine-9.6.1/common-bios/full.map.S' -o '/build/limine-9.6.1/common-bios/full.map.o'
Warning: supplying the --target i686-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-nostdlibinc' [-Wunused-command-line-argument]
rm -f '/build/limine-9.6.1/common-bios/full.map.S' '/build/limine-9.6.1/common-bios/full.map.d'
make -f common.mk '/build/limine-9.6.1/common-bios/empty'
make[3]: Entering directory '/build/limine-9.6.1/common'
make[3]: '/build/limine-9.6.1/common-bios/empty' is up to date.
make[3]: Leaving directory '/build/limine-9.6.1/common'
make -f common.mk '/build/limine-9.6.1/common-bios/linker.ld'
make[3]: Entering directory '/build/limine-9.6.1/common'
/nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p '/build/limine-9.6.1/common-bios'
clang -target i686-unknown-none-elf -target i686-unknown-none-elf -x c -E -P -undef linker_bios.ld.in -o '/build/limine-9.6.1/common-bios/linker.ld'
Warning: supplying the --target i686-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
make[3]: Leaving directory '/build/limine-9.6.1/common'
ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' '/build/limine-9.6.1/common-bios/full.map.o' -T'/build/limine-9.6.1/common-bios/linker.ld' -o '/build/limine-9.6.1/common-bios/limine.elf'
llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/build-id.s2.bin'
cd '/build/limine-9.6.1/common-bios' && \
        llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s2.bin build-id.s2.o && \
        llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s2.o
llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/build-id.s3.bin'
cd '/build/limine-9.6.1/common-bios' && \
        llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s3.bin build-id.s3.o && \
        llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s3.o
ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' '/build/limine-9.6.1/common-bios/full.map.o' '/build/limine-9.6.1/common-bios/build-id.s2.o' '/build/limine-9.6.1/common-bios/build-id.s3.o' -T'/build/limine-9.6.1/common-bios/linker.ld' -o '/build/limine-9.6.1/common-bios/limine.elf'
llvm-objcopy -O binary '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/limine-bios.sys'
chmod -x '/build/limine-9.6.1/common-bios/limine-bios.sys'
dd if='/build/limine-9.6.1/common-bios/limine-bios.sys' bs=$(( 0x$("llvm-readelf" -S '/build/limine-9.6.1/common-bios/limine.elf' | /nix/store/5ygilvgz6l47fw3x5ylb0cz1afgc3737-gnugrep-3.12/bin/grep '\.text\.stage3' | /nix/store/i74283mw5bncn16i0zbz0lvvq4sn0q87-gnused-4.9/bin/sed 's/^.*] //' | gawk '{print $3}' | /nix/store/i74283mw5bncn16i0zbz0lvvq4sn0q87-gnused-4.9/bin/sed 's/^0*//') - 0xf000 )) count=1 of='/build/limine-9.6.1/common-bios/stage2.bin' 2>/dev/null
gzip -n -9 < '/build/limine-9.6.1/common-bios/stage2.bin' > '/build/limine-9.6.1/common-bios/stage2.bin.gz'
make[2]: Leaving directory '/build/limine-9.6.1/common'
make[1]: Leaving directory '/build/limine-9.6.1'
make: *** [GNUmakefile:92: all] Error 2
error: builder for '/nix/store/l1bsw14bh2i51c9r1dw4asqawyjdawpk-limine-9.6.1.drv' failed with exit code 2;
       last 25 log lines:
       > make -f common.mk '/build/limine-9.6.1/common-bios/linker.ld'
       > make[3]: Entering directory '/build/limine-9.6.1/common'
       > /nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p '/build/limine-9.6.1/common-bios'
       > clang -target i686-unknown-none-elf -target i686-unknown-none-elf -x c -E -P -undef linker_bios.ld.in -o '/build/limine-9.6.1/common-bios/linker.ld'
       > Warning: supplying the --target i686-unknown-none-elf != x86_64-unknown-linux-gnu argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
       > clang: warning: argument unused during compilation: '-fstack-clash-protection' [-Wunused-command-line-argument]
       > clang: warning: argument unused during compilation: '--gcc-toolchain=/nix/store/qdknxw57cwy1jkrhq7fzmiis73j42jv6-gcc-14.3.0' [-Wunused-command-line-argument]
       > make[3]: Leaving directory '/build/limine-9.6.1/common'
       > ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' '/build/limine-9.6.1/common-bios/full.map.o' -T'/build/limine-9.6.1/common-bios/linker.ld' -o '/build/limine-9.6.1/common-bios/limine.elf'
       > llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/build-id.s2.bin'
       > cd '/build/limine-9.6.1/common-bios' && \
       >       llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s2.bin build-id.s2.o && \
       >         llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s2.o
       > llvm-objcopy -O binary --only-section=.note.gnu.build-id '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/build-id.s3.bin'
       > cd '/build/limine-9.6.1/common-bios' && \
       >  llvm-objcopy -I binary -B i386 -O elf32-i386 build-id.s3.bin build-id.s3.o && \
       >         llvm-objcopy --add-section .note.GNU-stack='/build/limine-9.6.1/common-bios/empty' --set-section-flags .note.GNU-stack=noload,readonly build-id.s3.o
       > ld.lld -nostdlib -z max-page-size=0x1000 --gc-sections -m elf_i386 -static --build-id=sha1 '/build/limine-9.6.1/common-bios/common/cc-runtime.s2.o' '/build/limine-9.6.1/common-bios/common/crypt/blake2b.o' '/build/limine-9.6.1/common-bios/common/drivers/disk.s2.o' '/build/limine-9.6.1/common-bios/common/drivers/edid.o' '/build/limine-9.6.1/common-bios/common/drivers/gop.o' '/build/limine-9.6.1/common-bios/common/drivers/serial.o' '/build/limine-9.6.1/common-bios/common/drivers/vbe.o' '/build/limine-9.6.1/common-bios/common/drivers/vga_textmode.o' '/build/limine-9.6.1/common-bios/common/entry.s2.o' '/build/limine-9.6.1/common-bios/common/entry.s3.o' '/build/limine-9.6.1/common-bios/common/fs/fat32.s2.o' '/build/limine-9.6.1/common-bios/common/fs/file.s2.o' '/build/limine-9.6.1/common-bios/common/fs/iso9660.s2.o' '/build/limine-9.6.1/common-bios/common/lib/acpi.o' '/build/limine-9.6.1/common-bios/common/lib/bli.o' '/build/limine-9.6.1/common-bios/common/lib/config.o' '/build/limine-9.6.1/common-bios/common/lib/elf.o' '/build/limine-9.6.1/common-bios/common/lib/elsewhere.o' '/build/limine-9.6.1/common-bios/common/lib/fb.o' '/build/limine-9.6.1/common-bios/common/lib/fdt.o' '/build/limine-9.6.1/common-bios/common/lib/getchar.o' '/build/limine-9.6.1/common-bios/common/lib/gterm.o' '/build/limine-9.6.1/common-bios/common/lib/guid.o' '/build/limine-9.6.1/common-bios/common/lib/image.o' '/build/limine-9.6.1/common-bios/common/lib/libc.o' '/build/limine-9.6.1/common-bios/common/lib/libc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/memory.s2.o' '/build/limine-9.6.1/common-bios/common/lib/misc.o' '/build/limine-9.6.1/common-bios/common/lib/misc.s2.o' '/build/limine-9.6.1/common-bios/common/lib/panic.s2.o' '/build/limine-9.6.1/common-bios/common/lib/part.s2.o' '/build/limine-9.6.1/common-bios/common/lib/pe.o' '/build/limine-9.6.1/common-bios/common/lib/print.s2.o' '/build/limine-9.6.1/common-bios/common/lib/rand.o' '/build/limine-9.6.1/common-bios/common/lib/stb_image.o' '/build/limine-9.6.1/common-bios/common/lib/term.o' '/build/limine-9.6.1/common-bios/common/lib/time.o' '/build/limine-9.6.1/common-bios/common/lib/trace.s2.o' '/build/limine-9.6.1/common-bios/common/lib/uri.o' '/build/limine-9.6.1/common-bios/common/menu.o' '/build/limine-9.6.1/common-bios/common/mm/mtrr.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.o' '/build/limine-9.6.1/common-bios/common/mm/pmm.s2.o' '/build/limine-9.6.1/common-bios/common/mm/vmm.o' '/build/limine-9.6.1/common-bios/common/protos/chainload.o' '/build/limine-9.6.1/common-bios/common/protos/limine.o' '/build/limine-9.6.1/common-bios/common/protos/linux.o' '/build/limine-9.6.1/common-bios/common/protos/linux_risc.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot1.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot2.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe.s2.o' '/build/limine-9.6.1/common-bios/common/pxe/tftp.s2.o' '/build/limine-9.6.1/common-bios/common/sys/a20.s2.o' '/build/limine-9.6.1/common-bios/common/sys/cpu_riscv.o' '/build/limine-9.6.1/common-bios/common/sys/e820.s2.o' '/build/limine-9.6.1/common-bios/common/sys/exceptions.s2.o' '/build/limine-9.6.1/common-bios/common/sys/gdt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/idt.o' '/build/limine-9.6.1/common-bios/common/sys/idt.s2.o' '/build/limine-9.6.1/common-bios/common/sys/lapic.o' '/build/limine-9.6.1/common-bios/common/sys/pic.o' '/build/limine-9.6.1/common-bios/common/sys/smp.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm.o' '/build/limine-9.6.1/common-bios/flanterm/src/flanterm_backends/fb.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_addresses.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_check.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_empty_tree.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_overlay.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_ro.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_rw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_strerror.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_sw.o' '/build/limine-9.6.1/common-bios/libfdt/src/fdt_wip.o' '/build/limine-9.6.1/common-bios/common/lib/mem.s2.o' '/build/limine-9.6.1/common-bios/common/menu_thunk.o' '/build/limine-9.6.1/common-bios/common/protos/limine_asm.o' '/build/limine-9.6.1/common-bios/common/sys/dummy_isr.o' '/build/limine-9.6.1/common-bios/common/entry_asm.s2.o' '/build/limine-9.6.1/common-bios/common/lib/real.s2.o' '/build/limine-9.6.1/common-bios/common/lib/sleep.o' '/build/limine-9.6.1/common-bios/common/lib/spinup.o' '/build/limine-9.6.1/common-bios/common/pxe/pxe_asm.s2.o' '/build/limine-9.6.1/common-bios/common/sys/int_thunks.s2.o' '/build/limine-9.6.1/common-bios/common/protos/limine_32.o' '/build/limine-9.6.1/common-bios/common/protos/linux_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_32.o' '/build/limine-9.6.1/common-bios/common/protos/multiboot_reloc.o' '/build/limine-9.6.1/common-bios/common/sys/smp_trampoline.o' '/build/limine-9.6.1/common-bios/stage2.map.o' '/build/limine-9.6.1/common-bios/full.map.o' '/build/limine-9.6.1/common-bios/build-id.s2.o' '/build/limine-9.6.1/common-bios/build-id.s3.o' -T'/build/limine-9.6.1/common-bios/linker.ld' -o '/build/limine-9.6.1/common-bios/limine.elf'
       > llvm-objcopy -O binary '/build/limine-9.6.1/common-bios/limine.elf' '/build/limine-9.6.1/common-bios/limine-bios.sys'
       > chmod -x '/build/limine-9.6.1/common-bios/limine-bios.sys'
       > dd if='/build/limine-9.6.1/common-bios/limine-bios.sys' bs=$(( 0x$("llvm-readelf" -S '/build/limine-9.6.1/common-bios/limine.elf' | /nix/store/5ygilvgz6l47fw3x5ylb0cz1afgc3737-gnugrep-3.12/bin/grep '\.text\.stage3' | /nix/store/i74283mw5bncn16i0zbz0lvvq4sn0q87-gnused-4.9/bin/sed 's/^.*] //' | gawk '{print $3}' | /nix/store/i74283mw5bncn16i0zbz0lvvq4sn0q87-gnused-4.9/bin/sed 's/^0*//') - 0xf000 )) count=1 of='/build/limine-9.6.1/common-bios/stage2.bin' 2>/dev/null
       > gzip -n -9 < '/build/limine-9.6.1/common-bios/stage2.bin' > '/build/limine-9.6.1/common-bios/stage2.bin.gz'
       > make[2]: Leaving directory '/build/limine-9.6.1/common'
       > make[1]: Leaving directory '/build/limine-9.6.1'
       > make: *** [GNUmakefile:92: all] Error 2
       For full logs, run:
         nix log /nix/store/l1bsw14bh2i51c9r1dw4asqawyjdawpk-limine-9.6.1.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.41, NixOS, 25.05 (Warbler), 25.05.20250815.ad7196a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - nixpkgs: `/nix/store/fbnlawcsvls3033n9ddi81grgw030881-source`


### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/li/limine/package.nix
+++ b/pkgs/by-name/li/limine/package.nix
@@ -23,6 +23,13 @@ let
     || (if targets == [ ] then stdenv.hostPlatform.isx86_64 else (builtins.elem "x86_64" targets))
     || enableAll;
 
+  missingZerocallusedregs =
+    (
+      if targets == [ ] then stdenv.hostPlatform.isLoongArch64 else (builtins.elem "loongarch64" targets)
+    )
+    || (if targets == [ ] then stdenv.hostPlatform.isRiscV64 else (builtins.elem "riscv64" targets))
+    || enableAll;
+
   biosSupport' = biosSupport && hasX86;
   pxeSupport' = pxeSupport && hasX86;
 
@@ -53,6 +60,10 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   enableParallelBuilding = true;
+
+  hardeningDisable = lib.optionals missingZerocallusedregs [
+    "zerocallusedregs"
+  ];
 
   nativeBuildInputs = [
     llvmPackages.libllvm

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -3329,6 +3329,8 @@ with pkgs;
     gtk3 = if stdenv.hostPlatform.isDarwin then gtk3-x11 else gtk3;
   };
 
+  limine-full = limine.override { enableAll = true; };
+
   liquidsoap = callPackage ../tools/audio/liquidsoap/full.nix {
     ffmpeg = ffmpeg_6-full;
     ocamlPackages = ocaml-ng.ocamlPackages_4_14;

